openstax/settings/base.py

```diff
@@ -435,6 +436,10 @@
 EVENTBRITE_API_PRIVATE_TOKEN = os.getenv('EVENTBRITE_API_PRIVATE_TOKEN', '')
```

Comment:
not for this PR - but it's possible we are not using Event Bright anymore and this key/secret can be removed